news/2025/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer.md

```diff
@@ -0,0 +1,291 @@
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>
+
+I played standard too, but I was just better at osu!catch, which made it more fun and motivating to grind.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Is there a specific memory from osu! that stands out to you? If so, which one?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>
+
+There are a lot of great memories, but the most memorable one for me was the Grand Finals match of the [osu! Catch The Beat World Cup 2014](/wiki/Tournaments/CWC/2014). Our opponent was Germany, a team full of outstanding players like [NoteKuroi](https://osu.ppy.sh/users/186642), [Nyan-Zapo](https://osu.ppy.sh/users/480676), and [DeathXHunter](https://osu.ppy.sh/users/405326).
+
+Due to a DDoS attack, we had to replay the match without a stream, and it was one of the most nerve-wracking moments of my osu! tournament career. But somehow, I performed better in the rematch and managed to win the World Cup for the first time. Even to this day, those three players from Germany remain some of my best rivals in the game, and I'll always be grateful to them for the unforgettable memories.
+
+---
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/17745759" style="background-image: url('/wiki/shared/avatars/-Isla-.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/17745759">-Isla-</a></p>
+
+Okay, now let's move on to mapping. What caused you to start mapping in osu!catch? Was there any particular reason or a person that motivated you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/702598" style="background-image: url('/wiki/shared/avatars/Spectator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/702598">Spectator</a></p>
+
+When I first started mapping, osu!catch specific maps used to be unrankable. I mostly tried mapping just for fun, and received some positive feedback from other players. There was an old forum for every osu!catch player and I used to share my very first maps [there](https://osu.ppy.sh/community/forums/topics/21061?n=3561).
+
+![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/forum_post.jpg)
```

Comment:
would it be too much to ask to re-take this screenshot without the lines at the top or bottom? the screenshot would fit in better without them

```suggestion
![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/forum_post.jpg)
```

Reply:
It's fine, will do xD

Reply:
Should be done